src/views/Hero/Hero.scss

```diff
@@ -1,11 +1,56 @@
+@use "src/stylesheets/typography";
+@use "src/stylesheets/responsiveness";
+
 .HeroView {
-  height: 100%;
+  display: flex;
+  overflow: hidden;
+  justify-content: center;
+  min-height: 100%;
 
   &__homeView {
     background: linear-gradient(180deg, var(--wedgewood__gradient) 0%, var(--waterleaf) 100%);
-    height: 90%;
+    min-height: 90%;
     border-bottom-left-radius: 50px;
     border-bottom-right-radius: 50px;
     box-shadow: 0 10px 16px rgba(0, 0, 0, 0.25);
   }
+
+  &__container {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: center;
+    width: 100%;
+    max-width: 1200px;
+    margin: 5em 3em;
+
+    @include responsiveness.respond(phone) {
+      margin: 1em;
+    }
+  }
+
+  &__visual {
+    // postcard styling
+    max-width: 500px;
+    margin: 0 auto;
+  }
+
+  &__text {
+    margin: 3em auto;
+
+    > h1 {
+      @include typography.h1;
+
+      > span {
+        color: var(--wedgewood__dark);
+      }
+    }
+
+    > p {
+      max-width: 375px;
+      margin: 2em 0;
+
+      @include typography.title-text;
+    }
```

Comment:
Lets stray away from doing this. Use classnames instead!

Reply:
Ahh, oops! My bad. 

Although, how would you go about making this nest only once? 

```css
.HeroView {
  &__container {
    // ...
    @include responsiveness.respond(phone) {
      // ...
    }
  }
```

Reply:
That's a really good point! How about we increate the nesting to two depth instead of 3?